### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Editor Window_Screen Reader: JAWS is not acknowledging the users regarding the options present for the quick help list items

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3380]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #JAWS 

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps:**
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
11) Tab till any tag in the editor and press alt + Enter to open quick help menu
12) Check if JAWS is giving proper description about the list items 

**Actual Result:**
JAWS is not acknowledging the users regarding the options present for the quick help list items

**Expected Result:**
JAWS should acknowledge the users that the list items are sub menu items 

**User Impact:**
Screen Reader users will not be provided with sufficient information

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=c1d83660-4891-4456-916d-4e23c4c6cd8a)

[Bug_3380_IntelliJ_Editor_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3461053/Bug_3380_IntelliJ_Editor_JAWS.pptx)

**Comment by andxu** [comment]
@soumyatippapuram The behavior of quick help on jsp is provided by Intellj, you can test the editing experience of jsp on a fresh install of Intellj without installing our plugins, create a maven-archetype-webapp project and edit src/main/webapp/index.jsp, JAWS does the same thing. Could you please file an external bug at  https://youtrack.jetbrains.com/issues?

**Comment by andxu** [comment]
Importance of experience: Little, the quick help menu is read by JAWS, only there are some sub menu items missing, user may use right arrow button to navigate to sub menu items
Quality of experience: Little, the quick help menu provides quick shortcut for some common action, but you still can input it manually.
Alternate solutions: use right arrow button to navigate to sub menu items or input it manually

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue which has been linked to AAT, please help us review it , thanks!

**Comment by soumyatippapuram** [comment]
Created a new issue https://youtrack.jetbrains.com/issue/IDEA-222349

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and this is non blocking scenario

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749774
